components/ILIAS/Authentication/classes/class.ilAuthPageEditorGUI.php

```diff
@@ -104,10 +120,15 @@ public function executeCommand(): void
         }
     }
 
-    private function forwardToPageObject(): void
+    private function forwardToPageObject(AuthPageEditorContext $context): void
     {
+        $request_value = match ($context) {
+            AuthPageEditorContext::LOGIN => 'loginpage_languages_key',
+            AuthPageEditorContext::LOGOUT => 'logoutpage_languages_key',
+        };
```

Comment:
Again, we could/ should think about moving this into the `Enum`, like the language variable retrieval.